plugins/gatsby-remark-variables/index.js

```diff
@@ -0,0 +1,68 @@
+const visit = require("unist-util-visit");
+
+function scopedEval(expr, context = {}) {
```

Comment:
based on some research from stackoverflow - might still be possible to leak scope but we're not allowing user input so hopefully its not a significant concern